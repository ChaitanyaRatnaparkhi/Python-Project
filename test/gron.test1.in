{
    "name": "John",
    "age": 30,
    "address": {
        "city": "New York",
        "zip": "10001"
    },
    "contacts": [
        {"type": "email", "value": "john@example.com"},
        {"type": "phone", "value": "555-1234"}
    ]
}