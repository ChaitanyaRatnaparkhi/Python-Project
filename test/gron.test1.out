json = {}
json.name = "John"
json.age = 30
json.address = {{}}
json.address.city = "New York"
json.address.zip = "10001"
json.contacts = []
json.contacts[0] = {}
json.contacts[0].type = "email"
json.contacts[0].value = "john@example.com"
json.contacts[1] = {}
json.contacts[1].type = "phone"
json.contacts[1].value = "555-1234"

